charts/move2kube/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: move2kube
+description: >
+  Helm chart to deploy the move2kube workflow. #magic___^_^___line
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.5
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.0.1"
```

Comment:
this should match move2kube version, at least the one installed by default (and validated). is this the case? (I see the move2kube-ui is installed as `latest`)

Reply:
updated to `latest`